planner/core/src/test/java/PlannerTest.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2014 SeaClouds
+ * Contact: SeaClouds
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import com.google.common.io.Resources;
+import eu.seaclouds.planner.matchmaker.Pair;
+import eu.seaclouds.platform.planner.core.HttpHelper;
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+@Test(groups = {"Live"})
+public class PlannerTest {
+
+    public void optimizerIntegration() throws Exception{
```

Comment:
What is suppose to test this method? Please add some javadoc to explain the intention as it is not clear.
As it is, it seems a OptimizerClientTest not a PlannerTest then.